### ISSUE TITLE
provide alt text/tooltips for icons on homepage

### ISSUE THREAD

**Issue by ramcq** [flatpak/flatpak.github.io#236]
The app and distro icons on the homepage aren't universally understandable to all (sighted) viewers and without a text equivalent, are not accessible to screen readers etc. Each icon should have an alt text which can be used as a tooltip.
